src/sql/workbench/browser/editor/profiler/profilerInput.ts

```diff
@@ -126,14 +130,40 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 	}
 
 	public override getName(): string {
-		let name: string = nls.localize('profilerInput.profiler', "Profiler");
+		let name: string = ProfilerInput.PROFILERNAME;
 		if (!this.connection) {
 			return name;
 		}
-		name += ': ' + this.connection.serverName.substring(0, 20);
+		if (this.connection.connectionName) {
+			name += ': ' + this.connection.connectionName.substring(0, 20);
+		}
+		else {
+			name += ': ' + this.connection.serverName.substring(0, 20);
+		}
 		return name;
 	}
 
+	public override getTitle(verbosity?: Verbosity): string {
```

Comment:
Similar question like before, where is this method called from?

Reply:
See above